src/ImuSensor.cc

```diff
@@ -166,6 +165,26 @@ bool ImuSensor::Load(const sdf::Sensor &_sdf)
     }
   }
 
+  std::string localization = _sdf.ImuSensor()->Localization();
+
+  if (localization == "ENU")
+  {
+    this->dataPtr->sensorOrientationRelativeTo = WorldFrameEnumType::ENU;
+  } else if (localization == "NED") {
+    this->dataPtr->sensorOrientationRelativeTo = WorldFrameEnumType::NED;
+  } else if (localization == "NWU") {
+    this->dataPtr->sensorOrientationRelativeTo = WorldFrameEnumType::NWU;
+  } else if (localization == "CUSTOM") {
+    this->dataPtr->sensorOrientationRelativeTo = WorldFrameEnumType::CUSTOM;
+  } else {
+    this->dataPtr->sensorOrientationRelativeTo = WorldFrameEnumType::NONE;
```

Comment:
I think a test case with localization unset may help improve coverage for this line

Reply:
Added the test case

Reply:
the test case doesn't actually cover this line because the `sdf::Imu` object has a default value of `"CUSTOM"` for the localization string:

* https://github.com/ignitionrobotics/sdformat/blob/sdf12/src/Imu.cc#L58
* https://github.com/ignitionrobotics/sdformat/blob/sdf12/sdf/1.7/imu.sdf#L6

I think we would need to set an unrecognized localization value in the SDFormat in order to reach this line of code

Reply:
Done